BeeTracker/app/src/main/java/com/blueprint/foe/beetracker/HomePageActivity.java

```diff
@@ -0,0 +1,46 @@
+package com.blueprint.foe.beetracker;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View.OnClickListener;
+import android.widget.Button;
+import android.app.Activity;
+import android.content.Intent;
+import android.view.Menu;
+import android.view.View;
+
+public class HomePageActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_home_page);
+        // Locate the button in activity_learn.xml
```

Comment:
nit: I think you can remove this comment since it's a pretty common thing in Android